.env

```diff
@@ -5,3 +5,4 @@ DB_PASS=postgres
 DB_PORT=5438
 DB_HOST=localhost
 ENABLE_SEQUELIZE_LOGGING: false
+JWT_SECRET=secret
```

Comment:
Would be nice to use environment

Reply:
This variable located in the .env file and accessed via process.end so I think I'm using environment here. Can you clarify, please?